cli/src/pcluster/api/pcluster_api.py

```diff
@@ -30,7 +30,7 @@
 from pcluster.models.compute_fleet_status_manager import ComputeFleetStatus
```

Comment:
Not part of this PR: we should probably get rid of this file since most if not all functionalities have been ported in the controllers classes.